python/ray/tests/test_client.py

```diff
@@ -331,16 +338,27 @@ def __init__(self):
 
             def say_hello(self, whom):
                 self.count += 1
-                return ("Hello " + whom, self.count)
+                return "Hello " + whom, self.count
+
+            @ray.method(num_returns=2)
+            def say_hi(self, whom):
+                self.count += 1
+                return "Hi " + whom, self.count
```

Comment:
why this is needed?

Reply:
Test `@ray.method(num_returns=2)` is handled correctly.